CMakeLists.txt

```diff
@@ -543,6 +544,7 @@ if (LWS_WITH_ZLIB)
 			foreach (lib ${LWS_LIBRARIES})
 				add_dependencies(${lib} zlib_internal)
 			endforeach()
+			list(APPEND LWS_LIBRARIES zlib_internal)
```

Comment:
I agree. Seems to be a left over from an earlier version of libwebsockets on which I had started to make all these changes some time ago.